shopsafe-frontend/src/app/components/feedback-forms/feedback-forms.component.css

```diff
@@ -0,0 +1,17 @@
+#title .heading-text {
+    padding-bottom: 20px;
+    text-align: center;
+    text-transform: uppercase;
```

Comment:
Ok I am so dumb but I didn't realize this affected the p as well -- maybe remove this again and just make the "Feedback Forms" text uppercase